code/modules/projectiles/guns/medbeam.dm

```diff
@@ -0,0 +1,114 @@
+/obj/item/weapon/gun/medbeam
+	name = "Medical Beamgun"
+	desc = "Delivers medical nanites in a focused beam."
```

Comment:
It literally causes a huge explosion if you cross the beams. That's a reference right there